### DISCUSSION TITLE
Locking Branches

### DISCUSSION THREAD

**Discussion by aljawaid** [JustFxDev/ThemeMaestro#11] [General]
@JustFxDev Hello, you seemed to have locked the `main` branch.  Please unlock it so I can push one minor CSS fix, then you can lock it back again.

I had already pushed my fix to the branch so it is stuck in a loop as the remote branch is locked.

**Comment by JustFxDev** [comment]
@aljawaid please check out if it works now

**Reply by aljawaid** [reply]
done, thank you
